models/concerns/validate_data.rb

```diff
@@ -484,6 +491,25 @@ def check_sticky_relation_consistency
     )
   end
 
+  def check_relation_compatibility_with_alternatives
+    not_handled_relations = []
+    @hash[:relations].each{ |relation|
+      next if Models::Relation::ALTERNATIVE_COMPATIBLE.include?(relation[:type])
+
+      services = @hash[:services].select{ |service| relation[:linked_service_ids]&.include?(service[:id]) }
+      not_handled_relations << relation if services.any?{ |service|
+        (service[:activities]&.size || 0) > 1
+      }
+    }
+
+    return unless not_handled_relations.any?
+
+    raise OptimizerWrapper::UnsupportedProblemError.new(
+      "The following relations contains alternatives but didn't handle them: ",
```

Comment:
The following relations are not compatible with alternative services or something like that?